package.json

```diff
@@ -77,7 +78,6 @@
     "ember-seamless-immutable-shim": "^1.0.0",
     "ember-sinon": "0.5.1",
     "ember-sortable": "^1.8.2",
-    "ember-tooltips": "0.7.0",
```

Comment:
bitHound was complaining about this being out-of-date and I didn't see any references to `tooltip` in the repo, so I just removed it.